llvm-external-projects/iree-dialects/test/Dialect/linalg_transform/tile-and-peel.mlir

```diff
@@ -11,11 +11,11 @@ func @matmul_tensors(
   // CHECK: scf.for {{.*}} to %[[c124]]
   // CHECK:   scf.for {{.*}} to %[[c128]]
   // CHECK:     scf.for {{.*}} to %[[c124]]
-  // CHECK:       linalg.matmul {{.*}} ins({{.*}} : tensor<4x4xf32>, tensor<4x4xf32>) outs({{.*}} : tensor<4x4xf32>) -> tensor<4x4xf32>
-  // CHECK:     linalg.matmul {{.*}} ins({{.*}} : tensor<4x3xf32>, tensor<3x4xf32>) outs({{.*}} : tensor<4x4xf32>) -> tensor<4x4xf32>
+  // CHECK:       linalg.matmul ins({{.*}} : tensor<4x4xf32>, tensor<4x4xf32>) outs({{.*}} : tensor<4x4xf32>) -> tensor<4x4xf32>
+  // CHECK:     linalg.matmul ins({{.*}} : tensor<4x3xf32>, tensor<3x4xf32>) outs({{.*}} : tensor<4x4xf32>) -> tensor<4x4xf32>
   // CHECK: scf.for {{.*}} to %[[c128]]
   // CHECK:   scf.for {{.*}} to %[[c127]]
-  // CHECK:     linalg.matmul {{.*}} ins({{.*}} : tensor<2x?xf32>, tensor<?x4xf32>) outs({{.*}} : tensor<2x4xf32>) -> tensor<2x4xf32>
+  // CHECK:     linalg.matmul ins({{.*}} : tensor<2x?xf32>, tensor<?x4xf32>) outs({{.*}} : tensor<2x4xf32>) -> tensor<2x4xf32>
```

Comment:
@nicolasvasilache could you help check if this is a reasonable change after integration? There are nothing between `linalg.matmul` and `ins`.

Reply:
the checks are reverted.